lib/common/domain/use_cases/get_env_use_case.dart

```diff
@@ -16,7 +16,7 @@ class GetEnvUseCase extends FutureUseCase<void, EnvEntity> {
   @override
   Future<EnvEntity> execute({dynamic param}) {
     return _configRepository.getEnv().then((env) {
-      logger().i("[GetEnvUseCase] Current env is: $env");
+      logger().d("[GetEnvUseCase] Current env is: $env");
```

Comment:
I don't wanna see this in logs, so we can lower print priority and switch our main logging priority in app to `Level.info`.